contracts/aggregator/common/variables.sol

```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+import './interfaces.sol';
+
+contract VariablesCommon {
+    address public owner;
+
+    uint256 internal ownerStatus;
+
+    bytes32 internal dataHash;
+    // if 1 then can enter flashlaon, if 2 then callback
+    uint256 internal status;
+
+    struct FlashloanVariables {
+        address[] _tokens;
+        uint256[] _amounts;
+        uint256[] _iniBals;
+        uint256[] _finBals;
+        uint256[] _instaFees;
+    }
+
+    mapping(uint256 => address) public routeToImpl;
+    mapping(uint256 => bool) public routeStatus;
+    mapping(uint256 => uint256) public routeToFee;
```

Comment:
What is this variable? Is it used anywhere?

Reply:
No, I was going to use mapping first but then added the `calculateFeeBPS` function in implementations. Forgot to remove it. Done!